packages/contracts-bedrock/src/L1/StandardValidator.sol

```diff
@@ -531,25 +637,48 @@ contract StandardValidator {
         return keccak256(bytes(_a)) == keccak256(bytes(_b));
     }
 
-    function validate(ValidationInput memory _input, bool _allowFailure) public view returns (string memory) {
+    function validate(ValidationInput memory _input, bool _allowFailure) external returns (string memory) {
+        return validate(
+            _input,
+            _allowFailure,
+            ValidationOverrides({ guardian: address(0), l1PAOMultisig: address(0), challenger: address(0) })
+        );
+    }
+
+    function validate(
+        ValidationInput memory _input,
+        bool _allowFailure,
+        ValidationOverrides memory _overrides
+    )
+        public
+        returns (string memory)
+    {
         string memory _errors = "";
 
         _errors = assertValidSuperchainConfig(_errors);
-        _errors = assertValidProxyAdmin(_errors, _input.proxyAdmin);
+        _errors = assertValidProxyAdmin(_errors, _input.proxyAdmin, _overrides);
         _errors = assertValidSystemConfig(_errors, _input.sysCfg, _input.proxyAdmin);
         _errors = assertValidL1CrossDomainMessenger(_errors, _input.sysCfg, _input.proxyAdmin);
         _errors = assertValidL1StandardBridge(_errors, _input.sysCfg, _input.proxyAdmin);
         _errors = assertValidOptimismMintableERC20Factory(_errors, _input.sysCfg, _input.proxyAdmin);
         _errors = assertValidL1ERC721Bridge(_errors, _input.sysCfg, _input.proxyAdmin);
         _errors = assertValidOptimismPortal(_errors, _input.sysCfg, _input.proxyAdmin);
-        _errors = assertValidDisputeGameFactory(_errors, _input.sysCfg, _input.proxyAdmin);
+        _errors = assertValidDisputeGameFactory(_errors, _input.sysCfg, _input.proxyAdmin, _overrides);
         _errors = assertValidPermissionedDisputeGame(
-            _errors, _input.sysCfg, _input.absolutePrestate, _input.l2ChainID, _input.proxyAdmin
+            _errors, _input.sysCfg, _input.absolutePrestate, _input.l2ChainID, _input.proxyAdmin, _overrides
         );
         _errors = assertValidPermissionlessDisputeGame(
-            _errors, _input.sysCfg, _input.absolutePrestate, _input.l2ChainID, _input.proxyAdmin
+            _errors, _input.sysCfg, _input.absolutePrestate, _input.l2ChainID, _input.proxyAdmin, _overrides
         );
 
+        string memory overridesString = getOverridesString();
+        if (bytes(overridesString).length > 0) {
+            if (bytes(_errors).length > 0) {
+                _errors = string.concat(overridesString, ",", _errors);
+            }
```

Comment:
Could we just do this? 

```suggestion
        string memory overridesString = getOverridesString();
        if (bytes(overridesString).length > 0) {
            _errors = string.concat(overridesString, ",", _errors);
        }
        clearOverrides();
```

We want to ensure that we always return the overrides string, whether or not there is an error.